ansible/roles/apache/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+# vars
+
+# Linux
```

Comment:
Why not combine vars and tasks in the same block?

Reply:
They would raise this error: `ERROR! conflicting action statements: include_vars, include_tasks`